### ISSUE TITLE
The Plague Lord/Pestilence (Brainstorming ideas) 

### ISSUE THREAD

**Issue by LunexKun** [TTT-2/TTT2#439]
This isn't finished yet, and only here to contain brainstormed ideas.

- The Plague Lord will spawn alone and in his own team and he has no allies.

- The Plague Lords aim is to infect all living players with the plague, once he achieves this goal he will turn into Pestilence, Horseman of the Apocolypse.

- To infect other players, the Plague Lord is able to place down infected phioles and infected Dead Bodies. If the phioles are stepped on, or destroyed while a player is near, the player/s become infected. If a player goes near an infected dead body, they become infected.

- If an infected player dies, their body will remein infected and share the same infectious trade as the bodies placed by the Plague Lord.

- If the Plague Lord kills a player, they will leave behind an infected body.

- The plague Lord can't choose his weapons and will not be able to pick up ammunition. He will start with a random primary and secondary weapon.

- Once transformed into Pestilence, the palyer can pick up any weapon and ammunition they want.

**Comment by Pythagorion** [comment]
has the Pestilence a special weapon?

**Comment by Lunethia** [comment]
All of the pestilences abilities and perks have yet to be decided.
Though I am highly in favour of a scythe like item that basically oneshots anyone in melee range.
I would also suggest to give him a full heal, bonus health and armor.

**Comment by Pythagorion** [comment]
When it comes to the Pestilence, he should be indeed more powerful than a normal Player. Even in his weakest shape, he should be stronger than normal players. The PL must do much to get there. He has to be a Hazard for everyone as Pestilence.

**Comment by Lunethia** [comment]
Yes, I agree. It is supposed to be a "payoff mechanic". You work towards your transformation and you are pretty weak before you get there. So it should be a satisfying feeling when you achieve your transformation. Basically, if players know that the Plague Lord is in the round, they should focus on taking him out early, just like you would do with the marker.

**Comment by Pythagorion** [comment]
Indeed. It reminds me of this game "Evolve". The players has one goal: To Stop the monster before it reaches Evolving Level 3 cause they know, it will be really strong. It must be like: When the Pestilence rises up, the game could end every moment. All the other player should know then: "Now we´ve got a huge problem." Talking about the transformation: Shall there be a notification like message or something, that announce the transformation of the Pestilence?

**Comment by TimGoll** [comment]
For the final fight only whitelisted weapons should be allowed. Otherwise one could simply kill him with a Thomas

**Comment by Pythagorion** [comment]
Disabling Shops or what do you mean?

**Comment by Lunethia** [comment]
@TimGoll  Maybe Pestilence becomes immune to anything but weapon damage? 
also @Pythagorion Yes, I was thinking about using the same banner that is used when you get your role at the start of the round. Maybe this time a black banner (as reference to the black death wink wink) which reads "Pestilence has Arrived", something like that. Something everyone CANNOT miss, something exciting and thrilling.
We could even implement a new character model indicating that it's him.

**Comment by TimGoll** [comment]
I'd say checking the weapon kind and either preventing the weapon usage or disable the damage

**Comment by Pythagorion** [comment]
We could do that, but Character Modelling isn´t my strength :D.

**Comment by Pythagorion** [comment]
And I have to say that I find the Name "Plague Lord" a little bit weird. For me it´s sounds like a metal band name

**Comment by Lunethia** [comment]
If you have a better suggestion that is not plaguebearer, get over here and tell me.
But in my opinion Plague Lord is pretty accurate even if it's a little extreme since he is trying to trancend to be pestilence he needs to be somewhat into his whole plague thing. It makes sense to name him like that, kinda.

**Comment by Pythagorion** [comment]
I haven´t a better one yet, but as the Pestilence is the final Destination the PL shouldn´t have a strong and powerful name like that. I mean Pestilence sounds really cool, but the question is, does it sound more powerful than Plague Lord? Names can be changed later on. I just think that the name should make the levels clear. The player must know by hearing or reading the name: PL is (just) phase one and Pestilence is the final one.

**Comment by Lunethia** [comment]
Okay, that seems reasonable. I can see where you are coming from and what the problem is.
And I already have an idea for a name.
What do you think about **The Plague Host** (Or just Plague Host)

**Comment by Pythagorion** [comment]
Yes, Something like that. Perhaps Plague Spreader or Plague Scout. Just to make clear that he is just the beginning

**Comment by Pythagorion** [comment]
Another idea would be lackey or servant. This would make the Pestilence more like a Master

**Comment by Lunethia** [comment]
Plagues Servant? Heck we could even make him a ritualist at this point. lol. Trying to summon his overlord.

**Comment by Pythagorion** [comment]
We could.

**Comment by Pythagorion** [comment]
But I think when we go with Something like ritualist, servant or lackey, it would be the best cuz it fits the most
